examples/tutorial.py

```diff
@@ -385,18 +423,22 @@ def smooth_in_block(output_group: str, block: daisy.Block):
             write_roi=block_write_roi,
             num_workers=5,
         )
-    ]
+    ],
+    multiprocessing=False,
 )
 
 # %%
-plt.imshow(zarr.open('sample_data.zarr', 'r')['smoothed_with_context'][:].transpose(1, 2, 0), origin="lower")
+plt.imshow(
+    zarr.open("sample_data.zarr", "r")["smoothed_with_context"][:].transpose(1, 2, 0),
+    origin="lower",
+)
 
 # %% [markdown]
 # Success! Notice that there is a fade to black at the border, due to the `fill_value=0` argument used when reading the data from the input Array.
 # Smoothing is poorly defined at the border of the volume - if you want different behavior, you can expand the input array to include extended data of your choice at the border, or shrink the total output roi by the context to only include the section of the output that depends on existing data.
 
 # %% [markdown]
-# ## Conclusion: Dask and Daisy
+# ###Conclusion: Dask and Daisy
```

Comment:
I would move the section about dask vs daisy between the "quickstart" (your part at the bottom that I think should go first) and the "in depth tutorial". This makes the flow make sense - we have some simple examples, these simple example can also be done with dask. However, daisy is good at X situations. If you have one of X situations and want to learn how to use daisy's unique capabilities to optimize your workflow, read the in-depth tutorial.

Reply:
I tried this but it felt pretty awkward. There's a lot of info needed to build up to the smoothing task, to then compare dask and daisy. I moved the quickstart examples up to the top, but left the dask comparison in the body